app/src/main/java/com/example/stromtracker/ui/raeume/raeumeErstellen/RaeumeErstellenFragment.kt

```diff
@@ -1,20 +1,29 @@
 package com.example.stromtracker.ui.raeume.raeumeErstellen
 
-
 import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.EditText
+import android.widget.*
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.ViewModelProvider
 import com.example.stromtracker.R
+import com.example.stromtracker.database.Haushalt
 import com.example.stromtracker.database.Raum
 import com.example.stromtracker.ui.raeume.RaeumeFragment
 import com.example.stromtracker.ui.raeume.RaeumeViewModel
+import com.google.android.material.navigation.NavigationView
 
 class RaeumeErstellenFragment : Fragment() {
     private lateinit var raeumeViewModel: RaeumeViewModel
+    private lateinit var raumnameneditfeld: EditText
+    private lateinit var savebutton: View
+    private lateinit var informationfield: TextView
+    private var raumnameleer: String =
+        "Raum kann nicht gespeichert werden, da Kein Name eingegeben wurde."
```

Comment:
Wieder Auslagerung in strings.xml

Reply:
Hat nicht funktioniert. Wollte ich schon machen